kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/subPages/SearchQuizFilters.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+
```

Comment:
Could we have a follow up issue for unifying these two components with the ones we have in lessons? I think that there are few differences to what we have in lessons and there is some non-trivial logic that would be good to have de-duplicated cc: @marcellamaki

Reply:
Yes! there will be some planned, incremental refactoring that we do that I'm planning for the 0.18 patch releases to further clean up and consolidate some logic, without getting into the weeds on everything right now. It's a good suggestion to do that for this code, @AlexVelezLl